styles/media-query.css

```diff
@@ -0,0 +1,263 @@
+@media screen and (min-width: 768px) {
+  body {
+    padding: 17px 120px;
+  }
+
+  .logo {
+    font-size: 15px;
+  }
+
+  .menu {
+    display: flex;
+    gap: 20px;
+  }
+
```

Comment:
- [ ] Please check my comment in the video 👇 

- Here is the [Feedback Video](https://www.youtube.com/watch?v=_EtSYq1w9WY) 😉